src/support/has.ts

```diff
@@ -173,7 +173,7 @@ add('es6-weakmap', () => {
 
 /* Miscellaneous features */
 add('microtasks', () => has('es6-promise') || has('host-node') || has('dom-mutationobserver'), true);
-add('postmessage', () => typeof global.postMessage === 'function', true);
+add('postmessage', () => typeof global.window !== 'undefined' && typeof global.postMessage === 'function', true);
```

Comment:
Not that it is a big deal, but it would be nice to add a comment here as to why we are doing the check this way, for future generations (think of the kids!), as it is non-obvious.